api/src/main/java/org/openmrs/Order.java

```diff
@@ -63,7 +63,7 @@
 		IN_PROGRESS,
 		EXCEPTION,
 		COMPLETED
-	};
```

Comment:
What improvement is this?

Reply:
A semicolon is a statement, which is doing nothing in this case. It was marked as uneccessary one per this rule: https://sonar.openmrs.org/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck